templates/base/footer.tmpl

```diff
@@ -13,6 +13,8 @@
 	{{template "base/footer_content" .}}
 {{if .RequireSimpleMDE}}
 	<script src="{{StaticUrlPrefix}}/js/easymde.js?v={{MD5 AppVer}}"></script>
+	<script src="{{StaticUrlPrefix}}/vendor/plugins/codemirror/addon/mode/loadmode.js"></script>
+	<script src="{{StaticUrlPrefix}}/vendor/plugins/codemirror/mode/meta.js"></script>
```

Comment:
Last I checked, Codemirror is not really webpack-able because it implements its own lazy-loading mechanism, so it'll probably need to stay vendored unless there is a webpack-friendly version.

Reply:
yes and I had left it mostly vendored. In particular I left the modeURL changing magic in just below this - so that should have continued to work.